exporters/otlp/otlp_metric_test.go

```diff
@@ -681,6 +682,9 @@ func runMetricExportTest(t *testing.T, exp *Exporter, rs []record, expected []me
 				continue
 			}
 			assert.ElementsMatch(t, ilm.GetMetrics(), g, "metrics did not match for:\n\tResource: %s\n\tInstrumentationLibrary: %s\n", k.resource, k.instrumentationLibrary)
+			// if diff := cmp.Diff(ilm.GetMetrics(), g); diff != "" {
```

Comment:
I was kind of confused when I saw this added because I had written almost exactly the same thing when I originally wrote the tests. Maybe we should just switch to `cmd.Diff` instead of `assert.ElementsMatch` permanently?

Reply:
I forgot about that. I think we should leave it in. I couldn't read the output prior to adding this.

Reply:
Oh! I remember why, now. I wanted to leave that in, but it's testing more than we want. It's an ordered test, unlike `ElementsMatch`. I can't figure out how to make an elements-match-independent-of-order test out of `gocmp`.